### DISCUSSION TITLE
Option for "wide" mode

### DISCUSSION THREAD

**Discussion by AdrianSosic** [pradyunsg/furo#747] [Ideas]
Hi, I wanted to know if there is any possibility to get rid of the margins on the left and right side, such that the center part uses more of the available space (see red arrows in screenshot below). Can these be somehow cleverly adjusted via a setting of the theme?

We've tried playing around with setting manual widths, but the way we did it totally messes up the mobile version of page:
![image](https://github.com/pradyunsg/furo/assets/23265127/4da0cd7b-1738-476b-ba64-a13275be60a1)

Some guidance would be very much appreciated! Also, if there is no such option yet, perhaps such a "wide mode" would be a useful enhancement of the theme?

<img width="1917" alt="image" src="https://github.com/pradyunsg/furo/assets/23265127/64caac83-6b83-4ea9-856e-93cf21cbd3c3">

**Comment by pavel-kirienko** [comment]
This override in custom CSS works fine, mobile version not affected:

```css
.page .sidebar-drawer { width: unset; }
.page .content { width: 100%; }
```

I think that the default width setting is unreasonably narrow. Unfortunately, it is more difficult to override it sensibly via the custom CSS instead of stretching everything full width; there is a note on this in the sources implying that it is desirable to prohibit this kind of customization, but I don't understand why:

https://github.com/pradyunsg/furo/blob/bca9dbb109ef2d01875486f1b964a96b81cad32d/src/furo/assets/styles/variables/_layout.scss#L3-L4

I wonder if @pradyunsg would accept a pull request that either makes the width configurable, or at least increases the default width from the current 46em to, say, 70em?

**Comment by herronelou** [comment]
This limit to the width, coupled with the overly large font used for the text has made it quite frustrating for some of our docs that use inline code snippet. Coe blocks are ok and use a smaller font, but inline code uses the same large font as the body and reaches the maximum width very quickly. 

So much for Furo being "Responsive --- adapting perfectly to the available screen space, to work on all sorts of devices."
